src/views/v2/month/day.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * View: Month View - Day
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/views/v2/month/day.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+// @todo: This is just for presentation purposes, while building the view.
+$day_number  = ( $week < 1 ) ? ( $day + 1 ) * ( $week + 1 ) : ( $day + 1 ) + $week * 7;
+
+$day_classes = [ 'tribe-events-calendar-month__day-date', 'tribe-common-h4' ];
+
+// @todo: check if we use classes here or if we wrap the day block directly with the classes (we also have `.tribe-events-calendar-month__day-date--current`).
+if ( $day_number == date( 'd', time() ) ) {
+	$day_classes[] = 'tribe-events-calendar-month__day-date--current';
+}
+?>
+<div class="<?php echo esc_attr( implode( ' ', $day_classes ) ); ?>" aria-label="Weekday, Month Daynumber">
+	<time><?php echo esc_html( $day_number ); ?></time>
```

Comment:
Let's be sure to add `datetime` attribute ; ) Also wonder if it would be appropriate to wrap this time tag in a heading (whatever is appropriate based on your current setup, maybe an `h4`?). Thoughts?

I also like how as part of the heading for a day, gcal also states how many events there are for the day: http://p.tri.be/WqsYUj. This would be a nice thing to add here as well (not as part of the time tag) and wrap that text in an a11y hidden span.

Last thought, if I remember right, the date for each cell will link off to the day view, correct? If so, we should make sure to add a link too : )

Reply:
> I also like how as part of the heading for a day, gcal also states how many events there are for the day: http://p.tri.be/WqsYUj. This would be a nice thing to add here as well (not as part of the time tag) and wrap that text in an a11y hidden span.

I'm not sure if I'm getting you on this one. What attribute is being used for that on the example you shared?

Reply:
Sorry about that, there is a lot here :P No attribute is being used, but as part of the heading, they nicely inform the user how many events are occurring on a given day. So we could do that as well `<span class="a11y visually hide">X events, </span><time datetime="">time</time>`